README.md

```diff
@@ -42,6 +42,17 @@ Build instructions and more details can be found in the
 * Step 1: Make a space flight simulator
 * Step 2: Bloat it with features
 
+## Compiling for Windows on Visual Studio
+You will need: cmake, Visual Studio
+* Clone the repo with `https://github.com/TheOpenSpaceProgram/osp-magnum`
+* Open the repo `cd osp-magnum`
+* Get submodules `git clone --recursive --shallow-submodules https://github.com/TheOpenSpaceProgram/osp-magnum`
+* Create the build folder `mkdir build && cd build`
+* Download [SDL2 visual studio development binaries](https://www.libsdl.org/release/SDL2-devel-2.0.12-VC.zip), and extract it into the build folder
+* Run the command `SET SDL2_DIR=SDL2-2.0.12 && cmake -DCMAKE_PREFIX_PATH=%SDL2_DIR% -DSDL2_LIBRARY_RELEASE=%SDL2_DIR%/lib/x64/*.lib -DSDL2_INCLUDE_DIR=%SDL2_DIR%/include ..` to create project files
+* Open the solution `OSP-MAGNUM.sln` in file explorer, and visual studio should open up
```

Comment:
If you make a PR that changes only this specific file, then we can merge it right away.

Reply:
Ima merge the commits real quick and we should be done

Reply:
Looks like you still have 6 commits in this PR.

Any luck with merging?

Reply:
Well, I tried rebasing, and nothing happened

Reply:
Looks like we have 5 commits, down from 6.

Just do this

git rebase -i origin/master
edit each line but one to say "fixup" instead of "pick"
save the file and exit the editor

then check "git log" to confirm the rebase applied successfully.

Reply:
I think my main branch is pretty screwed up, I'll rebase it to master